war/pom.xml

```diff
@@ -436,6 +432,12 @@ THE SOFTWARE.
                   <version>1.6.2-5</version>
                   <type>hpi</type>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.jenkins-ci.modules</groupId>
+                  <artifactId>instance-identity</artifactId>
+                  <version>3.0.1</version>
```

Comment:
I think we want https://github.com/jenkinsci/instance-identity-plugin/pull/21 released first. 3.0 & 3.0.1 should be considered broken.

Reply:
A request made without reasoning. I will not accommodate such a request unless reasoning is provided.